src/cljs/athens/views/blocks/core.cljs

```diff
@@ -55,10 +55,15 @@
                                 :pointer-events "none"
                                 :border-radius "0.25rem"
                                 :transition "opacity 0.075s ease"
-                                :background (style/color :link-color :opacity-lower)
-                                :box-shadow [["0 0.25rem 0.5rem -0.25rem" (style/color :background-color :opacity-med)]]}]
+                                :background (style/color :link-color :opacity-lower)}]
                      [:&.is-selected:after {:opacity 1}]
-                     [:.block-body {:display "flex"
+                     [:.block-body {:display "grid"
+                                    :grid-template-columns "1em 1em 1fr auto"
+                                    :grid-template-rows "0 1fr 0"
+                                    :grid-template-areas "
+                                      'above above above above'
+                                      'toggle bullet content refs'
+                                      'below below below below'"
```

Comment:
Defining a grid here is what made most of this possible, so we don't have to leave elements in as placeholders.